truffle.js

```diff
@@ -21,8 +21,10 @@ const config = {
     test: {
       provider: Ganache.provider({
         unlocked_accounts: [0, 1, 2, 3, 4, 5],
-        vmErrorsOnRPCResponse: false,
-        default_balance_ether: 500
+        vmErrorsOnRPCResponse: true,
```

Comment:
should we user vmErrorsOnRPCResponse: true now? or need to change it back?

Reply:
I think we should turn it on now, since starting from ganache-core  v2.2.0 (https://github.com/trufflesuite/ganache-core/releases/tag/v2.2.0) we have `revert reason string` in result error message